worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,12 @@
 import math
-import random
+import numpy
 
 
-def random_point(width, height):
-    return random.randrange(0, width), random.randrange(0, height)
+def random_point(width, height, randint_substitute=None):
```

Comment:
I just didn't see the function being called with the extra param, so I questioned its utility. :)

If you intend on using it elsewhere in the code, then no problem.